adafruit_bmp3xx.py

```diff
@@ -205,15 +211,15 @@ def _read_coefficients(self):
             coeff[13] / 2**65.0,
         )  # P11
 
-    def _read_byte(self, register):
+    def _read_byte(self, register: int) -> int:
         """Read a byte register value and return it"""
         return self._read_register(register, 1)[0]
 
-    def _read_register(self, register, length):
+    def _read_register(self, register: int, length: int) -> None:
```

Comment:
Since this actually raises `NotImplementedError` with the intention that subclasses define it, it's actually useful to annotate with the types that said implementations *should* be using.  Si for this, let's type annotate with a return of `bytearray`.